worklets/Overview.bs

```diff
@@ -449,14 +449,16 @@ When the user agent is to <dfn>fetch and invoke a worklet script</dfn> given |wo
 
             2. Reject |promise| with an "{{AbortError}}" {{DOMException}}.
 
-    4. If |script|'s <a>error to rethrow</a> is not null, then run these steps:
+    4. If |script|'s <a>error to rethrow</a> is not null, then <a>queue a task</a> on
+        |outsideSettings|'s <a>responsible event loop</a> given |script|'s <a>error to rethrow</a>
+        to run these steps:
 
         1. If |pendingTaskStruct|'s <a for="pending tasks struct">counter</a> is not <b>-1</b>, then
             run these steps:
 
             1. Set |pendingTaskStruct|'s <a for="pending tasks struct">counter</a> to <b>-1</b>.
 
-            2. Reject |promise| with |script|'s <a>error to rethrow</a>.
```

Comment:
I assume the intention of this change is to reference the local argument passed to the task (i.e. avoid dereferencing |script| here), but `<a>error to rethrow</a>` still references `#concept-script`'s `error to rethrow`.